python/cugraph/setup.py

```diff
@@ -131,49 +67,17 @@ def run(self):
         os.system('find . -name "*.cpython*.so" -type f -delete')
 
 
-class build_ext_no_debug(_build_ext):
-
-    def build_extensions(self):
-        def remove_flags(compiler, *flags):
-            for flag in flags:
-                try:
-                    compiler.compiler_so = list(
-                        filter((flag).__ne__, compiler.compiler_so)
-                    )
-                except Exception:
-                    pass
-        # Full optimization
-        self.compiler.compiler_so.append("-O3")
-        # No debug symbols, full optimization, no '-Wstrict-prototypes' warning
-        remove_flags(
-            self.compiler, "-g", "-G", "-O1", "-O2", "-Wstrict-prototypes"
-        )
-        super().build_extensions()
-
-    def finalize_options(self):
-        if self.distribution.ext_modules:
-            # Delay import this to allow for Cython-less installs
-            from Cython.Build.Dependencies import cythonize
-
-            nthreads = getattr(self, "parallel", None)  # -j option in Py3.5+
-            nthreads = int(nthreads) if nthreads else None
-            self.distribution.ext_modules = cythonize(
-                self.distribution.ext_modules,
-                nthreads=nthreads,
-                force=self.force,
-                gdb_debug=False,
-                compiler_directives=dict(
-                    profile=False, language_level=3, embedsignature=True
-                ),
-            )
-        # Skip calling super() and jump straight to setuptools
-        setuptools.command.build_ext.build_ext.finalize_options(self)
-
-
 cmdclass = dict()
 cmdclass.update(versioneer.get_cmdclass())
-cmdclass["build_ext"] = build_ext_no_debug
 cmdclass["clean"] = CleanCommand
```

Comment:
I'm not 100% sure, but scikit-build's clean should do a pretty good job cleaning out everything. The main potential exception is that I don't know if it wipes out cpp/so files inside the source directory if you do an inplace build. If you all really want that then perhaps it's worth the override. I don't think any of the other RAPIDS libs are bothering to do so, though.